src/routes/admin/kitchen/OrderList.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+  import type { State } from "$lib/types";
+  interface Props {
+    // this is a custom store Order[], I am not sure how to type it.
+    orders: any;
+    show: State;
+    state: string;
+  }
+
+  let { orders, show, state }: Props = $props();
+</script>
+
+<div class="overflow-y-auto">
+  <ul class="flex-col">
+    {#each $orders as order}
+      {#if show.includes(order.state)}
+        <li>
+          <!-- maybe on hover/click (if click, swipe to delete) we can show the contents? -->
+          <button
+            class="btn mt-4 w-full text-xl font-normal md:btn-lg md:text-3xl"
+            onclick={() => orders.setState(order.id, state)}
+          >
+            {order.id.slice(0, 4)}
```

Comment:
Bruke indeksen til order for å lage økende ID, heller enn å bruke ID fra pocketbase direkte. Starter på 100 for at det skal være lik mengde siffer på alle. Dette virke, men det forutsetter at det ikke er for mange orders i listen over orders, så det vil trenge en mekanisme for å nullstille.
```suggestion
    {#each $orders as order, index}
      {#if show.includes(order.state)}
        <li>
          <!-- maybe on hover/click (if click, swipe to delete) we can show the contents? -->
          <button
            class="btn mt-4 w-full text-xl font-normal md:btn-lg md:text-3xl"
            onclick={() => orders.setState(order.id, state)}
          >
            {index + 100}
```

Reply:
grei ide, vi kan starte med dette og komme på en ny løsning hvis situasjonen krever det.

Hehe, dette minte meg litt om leftpad nå egentlig. [[1](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)] [[2](https://en.wikipedia.org/wiki/Npm_left-pad_incident)]